pkg/image/api/validation/validation.go

```diff
@@ -75,10 +75,6 @@ func ValidateImageStream(stream *api.ImageStream) field.ErrorList {
 		result = append(result, field.Invalid(field.NewPath("metadata", "name"), stream.Name, fmt.Sprintf("'namespace/name' cannot be longer than %d characters", reference.NameTotalLengthMax)))
 	}
 
-	if stream.Spec.Tags == nil {
```

Comment:
I don't believe we have defaulting for this elsewhere, so removing this without any other changes may cause issues